tests/components/upnp/test_sensor.py

```diff
@@ -93,16 +96,18 @@ async def test_derived_upnp_sensors(
     now = coordinator.data[TIMESTAMP]
     with patch(
         "homeassistant.components.upnp.device.utcnow",
-        return_value=now + UPDATE_INTERVAL,
+        return_value=now + timedelta(seconds=DEFAULT_SCAN_INTERVAL),
```

Comment:
I'd update `now` to use `dt_util.utcnow()` and then use `now` below.

Reply:
Fixed.
We should get `now` from `coordinator.data[TIMESTAMP]` - it's the time just before the coordinator was setup - to ensure the bumped time triggers an update.

Reply:
I don't agree. It's enough to set up the integration first so that the first refresh happens and then take the current time and move it forward beyond the next update by adding the update interval.

Reply:
You're absolutely, right, my bad 👍